e2e/production-e2e/Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM python:3.8.2-slim-buster as base
+# syntax=docker/dockerfile:1
 
```

Comment:
Could you please add maintenance information for this docker file?

Reply:
Is it okay to use info@hydrosphere.io as a maintainer?

Reply:
support@hydrosphere.io much better